worlds/sc2/client.py

```diff
@@ -1044,6 +1045,21 @@ def kerrigan_primal(ctx: SC2Context, kerrigan_level: int) -> bool:
         return get_full_item_list()[item_names.KERRIGAN_PRIMAL_FORM].code in codes
     return False
 
+
+def get_mission_variant(mission_id):
+    mission_flags = lookup_id_to_mission[mission_id].flags
+    faction_variant = 0
+    if MissionFlag.RaceSwap not in mission_flags:
+        return faction_variant
+    if MissionFlag.Terran in mission_flags:
+        faction_variant = 1
+    elif MissionFlag.Zerg in mission_flags:
+        faction_variant = 2
+    elif MissionFlag.Protoss in mission_flags:
+        faction_variant = 3
+    return faction_variant
```

Comment:
Recommend some type annotations, and putting the `return` statements in consistent places:
```python
def get_mission_variant(mission_id: int) -> int:
    mission_flags = lookup_id_to_mission[mission_id].flags
    if MissionFlag.RaceSwap not in mission_flags:
        return 0
    if MissionFlag.Terran in mission_flags:
        return 1
    elif MissionFlag.Zerg in mission_flags:
        return 2
    elif MissionFlag.Protoss in mission_flags:
        return 3
```